### PR TITLE
Update defi.tokenlist.json

### PR DESCRIPTION
**Pull request by DrZee3**
Add ICEDOGE token

### CODE REVIEW

File: defi.tokenlist.json [hunk 1]
```diff
@@ -563,6 +563,14 @@
           "symbol": "JENKEM",
           "decimals": 18,
           "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0xdE61d83F83950fDe967BD2C68B1a9f64694F85Ff/logo.png"
+        }
+                {
```
Comment (SarjuHansaliya) [inline defi.tokenlist.json:567]: 1. invalid json at 566th line
2. needs proper formatting at 567th line

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

**Comment by SarjuHansaliya** [comment]
@DrZee3 can you please resolve conflicts in this PR ?

File: defi.tokenlist.json [hunk 2]
```diff
@@ -595,6 +602,7 @@
           "symbol": "RAI",
           "decimals": 18,
           "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0x97Cd1CFE2ed5712660bb6c14053C0EcB031Bff7d/logo.png"
+>>>>>>> main
```
Comment (bmino) [inline defi.tokenlist.json:605]: This shouldn't be here. Looks like it's left behind from some previous merge conflict resolution?

File: defi.tokenlist.json [hunk 3]
```diff
@@ -566,6 +566,13 @@
         },
         {
           "chainId": 43114,
+
+          "address": "0xBf9D7eCC6D5f09B9a10b8a39921E6f9cbA3c5D48",
+          "name": "ICEDOGE",
+          "symbol": "ICEDOGE",
+          "decimals": 9,
+          "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0xBf9D7eCC6D5f09B9a10b8a39921E6f9cbA3c5D48/logo.png"
+        },
```
Comment (bmino) [inline defi.tokenlist.json:575]: This is now invalid JSON missing an opening bracket

File: defi.tokenlist.json [hunk 4]
```diff
@@ -566,6 +566,13 @@
         },
         {
           "chainId": 43114,
+
```
Comment (bmino) [inline defi.tokenlist.json:569]: Please remove this line break

**Review by bmino — CHANGES_REQUESTED** [review]

**Comment by bmino** [comment]
I think it's missing a chainId too

**Comment by bmino** [comment]
Formatting looks good now. You'll need to update the minor version and timestamp as per the [README](https://github.com/pangolindex/tokenlists#adding-your-token)

**Comment by DrZee3** [comment]
> Formatting looks good now. You'll need to update the minor version and timestamp as per the [README](https://github.com/pangolindex/tokenlists#adding-your-token)

thank you bmino. just updated, please check. hope its fine

**Review by bmino — APPROVED** [review]

**Review by SarjuHansaliya — APPROVED** [review]

**Pull request merged**